api/src/plugins/causemos.py

```diff
@@ -184,6 +179,12 @@ def to_parameter(annot):
             "max": float(annot["max"]) if annot["max"] != "" else None,
         }
 
+
+def convert_to_causemos_format(model):
```

Comment:
This had a reference to `self` that would not work, since it isn't part of a class and was expecting 2 params, receiving one when called.